### PR TITLE
Pm2020

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/chSDcP3i/39-i-want-a-proposal-for-an-oddbird-project-manager

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- site-wide top nav
- /jobs/

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[I want a proposal for an OddBird Project Manager.](https://trello.com/c/chSDcP3i/39-i-want-a-proposal-for-an-oddbird-project-manager)

File: content/jobs.md [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+## We’re Looking for Someone Who:
+
+- Has knowledge of the web design and development process
+- Can manage multiple projects simultaneously
+- Will help plan, monitor, and facilitate projects on time and within budget 
+- Is an excellent verbal and written communicator
+- Is not tied to a single methodology, 
+  but can help guide the team in finding & implementing 
+  processes that work for us
+- Is located in a US time-zone
```
Comment (carljm) [inline content/jobs.md:23]: Technically this includes, like, Hawaii and American Samoa  and Guam :) Would it be both more accurate and less US-centric to say e.g. "is located in a continental North America timezone"?

**Review by SondraE — APPROVED** [review]
Looking great! 
The only thing I can think of is maybe we could use a version of our logo in the header of this post that doesn't have the word "OddBird". It's a bit busy with the logo in the header so close to the same logo in the upper left of the page.

**Review by carljm — APPROVED** [review]
Looks reasonable to me! The lack of a specified (initial) hourly rate stands out -- did we also not include one in our previous job postings? I would guess that's something people will want to know at some point. Have you all already talked about rate? I assume the plan is to pay the PM same as the devs? Or will more of their work be classified as "internal" thus likely resulting in a lower overall rate?

**Review by jgerigmeyer — APPROVED** [review]

**Comment by mirisuzanne** [comment]
> The lack of a specified (initial) hourly rate stands out -- did we also not include one in our previous job postings? I would guess that's something people will want to know at some point. Have you all already talked about rate? I assume the plan is to pay the PM same as the devs? Or will more of their work be classified as "internal" thus likely resulting in a lower overall rate?

@carljm I agree it stands out, but we've never included it before, and the thought was that it's hard to summarize. Yes, the plan is that it's paid the same, but I expect their work will fall across all three rates we currently have - and also might raise some questions about the two internal rates. So what rate do we list exactly? Open to other ideas on how we handle that, or convey it more clearly.

File: content/jobs.md [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+  - Manage scope
+  - Review completed, in progress, and upcoming items
+- Prioritize tasks:
+  - Assign current sprint based on client’s goals and team availability
+  - Set milestones
+- Identify potential risks and work to avoid project blockers
+- Monitor project timelines and budgets
+- Ensure team is clear on what to work on and when it is due
+
+## At OddBird We:
```
Comment (wlonk) [inline content/jobs.md:39]:
```suggestion
## At OddBird, We:
```

**Review by wlonk — APPROVED** [review]

File: content/jobs.md [hunk 3]
```diff
@@ -1,6 +1,89 @@
...
+We're also not a start-up
+expecting you to work nights and weekends towards future equity.
+This is a contract job with long term potential,
+a good hourly rate, a collective vision, and a lot of flexibility.
+We work together to find clients,
+and we work together to build them the best tools we can.
+When there's work to be done,
+we all do what we can,
+and we get paid for our time.
+
```
Comment (duggiemitchell) [inline content/jobs.md:78]: <3 this!

**Review by duggiemitchell — APPROVED** [review]
Looks great! can't wait to meet them 👍🏼

**Pull request merged**